app/src/shared/components/ActivityList/index.jsx

```diff
@@ -0,0 +1,118 @@
+// @flow
+
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { useSelector, useDispatch } from 'react-redux'
+
+import Activity, { actionTypes } from '$shared/utils/Activity'
+import { getFilters } from '$userpages/utils/constants'
+import { selectStreams } from '$userpages/modules/userPageStreams/selectors'
+import { getStreams } from '$userpages/modules/userPageStreams/actions'
+import { selectMyProductList } from '$mp/modules/myProductList/selectors'
+import { getMyProducts } from '$mp/modules/myProductList/actions'
+import { selectCanvases } from '$userpages/modules/canvas/selectors'
+import { getCanvases } from '$userpages/modules/canvas/actions'
+
+import ActivityListItem from './ActivityListItem'
+
+const Container = styled.div`
+    width: 400px;
+    height: 560px;
+    display: flex;
+    flex-direction: column;
+`
+
+const Tabs = styled.div`
+    display: flex;
+    width: 100%;
+    padding: 16px 0 0 16px;
+`
+
+const Tab = styled.div`
```

Comment:
Looks like tab's underline need some love. 😃 

<img width="958" alt="Screenshot 2020-04-14 13 06 54" src="https://user-images.githubusercontent.com/320066/79220154-caeaeb00-7e53-11ea-892f-4fd0aad64776.png">

Reply:
↑ design on the left.